src/ssl.c

```diff
@@ -9430,6 +9433,10 @@ int wolfSSL_CertManagerLoadCA(WOLFSSL_CERT_MANAGER* cm, const char* file,
         return ret;
     }
 
+    /* Some configurations like OPENSSL_COMPATIBLE_DEFAULTS may turn off
+     * verification by default. Let's restore our desired defaults. */
+    wolfSSL_CTX_set_verify(tmp, WOLFSSL_VERIFY_DEFAULT, NULL);
```

Comment:
We are using a temporary WOLFSSL_CTX to load certificate.
Are the defaults not set by default?

Reply:
When `OPENSSL_COMPATIBLE_DEFAULTS` is defined, then verification gets turned off in `wolfSSL_CTX_new_ex`. I left this in for all configurations to make sure that nothing turns off verification as that is what we want in this function.